### PR TITLE
Bugfixes and Decoder + Merge

### PR DESCRIPTION
**Pull request by beppo-sturmtief**
Hi Lode,

i added some fixes and a test for the columnar batch encoding/decoding.
Serialization and compaction of Schema and Batches still open.

Best regards,
Simon

### CODE REVIEW

File: columnar_batch/columnar_batch.h [hunk 1]
```diff
@@ -1,6 +1,9 @@
 #include "../fusion_power_video.h"
 
-namespace fpvc {
+#ifndef COLUMNAR_BATCH_H_
```
Comment (lvandeve) [inline columnar_batch/columnar_batch.h]: Does this also work with the include guards starting at the very top above the include?

File: columnar_batch/columnar_batch_encoder.h [hunk 2]
```diff
@@ -5,7 +5,11 @@
 #include "columnar_batch.h"
 #include "../fusion_power_video.h"
 
-namespace fpvc {
+
+#ifndef COLUMNAR_BATCH_ENCODER_H_
```
Comment (lvandeve) [inline columnar_batch/columnar_batch_encoder.h]: here as well, can the include guard go around everything?

**Comment by lvandeve** [comment]
Thanks! I just left a few small remarks

**Comment by beppo-sturmtief** [comment]
Thanks for checking thoroughly - fixed!
Bests, Simon

**Comment by beppo-sturmtief** [comment]
Thanks for checking thoroughly - fixed!
Bests, Simon

**Comment by lvandeve** [comment]
Thanks!

**Pull request merged**